CI/Style-To-Repo/deploy.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Style to Repo
+#
+# Copyright (c) 1998-2009 ILIAS open source, Extended GPL, see docs/LICENSE
+#
+# Author Daniel Weise <daniel.weise@concepts-and-training.de>
+#
+# This script compares the actual style repo with the built style folder and pushes the possible changes to repo.
+
+REPO="git@github.com:daniwe4/style_test.git"
+NOW=$(date +'%d.%m.%Y %I:%M:%S')
+BASE_FOLDER="./CI/Style-To-Repo/repo"
+
+if [ -d ${BASE_FOLDER} ]
+then
+  rm -rf ${BASE_FOLDER}
+fi
+
+mkdir -p ${BASE_FOLDER}
+
+git clone ${REPO} ${BASE_FOLDER} >/dev/null 2>&1
+
+rm -rf ${BASE_FOLDER}/*
+
+cp -r CI/Style-To-Repo/style/* ${BASE_FOLDER}
+
+git -C ${BASE_FOLDER} update-index --really-refresh >/dev/null 2>&1
+git -C ${BASE_FOLDER} diff-index --quiet HEAD
+
+CHECK=$?
+if [[ "${CHECK}" == "0" ]]
+then
+  echo "[${NOW}] No changes detected on style files."
+else
+  echo "[${NOW}] Detect changes on style files. They will be committed to ${REPO}"
+  git -C ${BASE_FOLDER} add . >/dev/null 2>&1
+  git -C ${BASE_FOLDER} commit -m "[${NOW}] Detect changes on style files." >/dev/null 2>&1
+  git -C ${BASE_FOLDER} push origin master >/dev/null 2>&1
```

Comment:
Please use a branch (as base and as target) that has the very same name as the branch that the commit in the ILIAS repo was made on. If that branch does not exist on remote, it should be created.